Description: Add "arcade" sub-menu, and German translation
Origin: other, http://bugs.debian.org/478974
Origin: other, http://bugs.debian.org/641822

--- xmoto-0.5.9.orig/extra/xmoto.desktop
+++ xmoto-0.5.9/extra/xmoto.desktop
@@ -3,7 +3,8 @@
 Version=1.0
 Name=Xmoto
 GenericName=Motocross game
+GenericName[de]=Motocross-Spiel
 GenericName[fr_FR]=Jeu de moto
 Icon=xmoto.xpm
 Exec=xmoto
-Categories=Game;SportsGame;
+Categories=Game;SportsGame;ArcadeGame;
