Index: xmoto-0.4.2/src/states/StateScene.cpp
===================================================================
--- xmoto-0.4.2.orig/src/states/StateScene.cpp	2008-10-11 09:16:03.000000000 -0400
+++ xmoto-0.4.2/src/states/StateScene.cpp	2008-10-11 09:17:46.000000000 -0400
@@ -540,7 +540,7 @@
       int v_nbPlayed, v_nbDied, v_nbCompleted, v_nbRestart, v_playedTime;
       std::string v_idLevel = m_universe->getScenes()[0]->getLevelSrc()->Id();
       
-      v_result = v_pDb->readDB("SELECT nbPlayed, nbDied, nbCompleted, nbRestarted, playedTime "
+      v_result = v_pDb->readDB("SELECT IFNULL(nbPlayed,0), IFNULL(nbDied,0), IFNULL(nbCompleted,0), IFNULL(nbRestarted,0), IFNULL(playedTime,0) "
 			       "FROM stats_profiles_levels "
 			       "WHERE id_profile=\"" + xmDatabase::protectString(XMSession::instance()->profile()) + "\" "
 			       "AND id_level=\""     + xmDatabase::protectString(v_idLevel) + "\";",
