Description: Ensure we use system libraries
Author: Stephen Kitt <skitt@debian.org>

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -85,13 +85,8 @@
   MS_MKDIR
 )
 
-add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/bzip2" bzip2)
-add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/libccd" libccd)
 add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/chipmunk" chipmunk)
-add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/lua" lua)
 add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/md5sum" md5sum)
-add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/ode" ode)
-add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/xdgbasedir" xdgbasedir)
 add_subdirectory("${PROJECT_SOURCE_DIR}/vendor/glad" glad)
 
 configure_file(common/XMBuildConfig.h.in common/XMBuildConfig.h)
--- a/extra/CMakeLists.txt
+++ b/extra/CMakeLists.txt
@@ -13,6 +13,6 @@
     FILES
       xmoto.bmp
       xmoto.ico
-    DESTINATION share/xmoto
+    DESTINATION share/games/xmoto
   )
 endif()
